test/watcher.js

```diff
@@ -74,16 +75,11 @@ test('chokidar is installed', function (_t) {
 
 		api.run.reset();
 		api.run.returns(new Promise(function () {}));
-		api.files = [
+		files = [
 			'test.js',
 			'test-*.js',
 			'test'
 		];
-		api.excludePatterns = [
-			'!**/node_modules/**',
-			'!**/fixtures/**',
-			'!**/helpers/**'
-		];
```

Comment:
I suppose technically this doesn't have to be restored before each test, but it won't do any harm either. If you do want to move it you should add it to the `api` declaration <https://github.com/sotojuan/ava/blob/1d6d84442b99407ae0c329a48c75125ad6a1e4f9/test/watcher.js#L42:L44>